mtbf_driver/tests/mtbf/browser/test_mtbf_browser_clear_browsing_history.py

```diff
@@ -48,5 +48,6 @@ def test_browser_clear_history(self):
         self.assertEqual(0, search.history_items_count)
 
     def tearDown(self):
+        self.apps.kill(self.settings.app)
```

Comment:
we cannot just leave the settings app opened...it will break the next trial on opening any settings in setting app....   I can try return to home button...hopefully it will return to the init state of setting app.